SAMtags.tex

```diff
@@ -453,6 +455,112 @@ \subsubsection{Color space}
+field, one of {\tt A}, {\tt C}, {\tt G}, {\tt T} or {\tt N}, with
+the exception that {\tt N} is used to match any base rather than
+strictly {\tt N}.  This is followed by either plus or minus
+indicating the strand the modification was observed on (relative to
+the original sequenced strand of {\sf SEQ} with plus meaning same
+orientation\footnote{Hence a tool that may reverse complement sequences does not need to understand how to manipulate the {\tt MM} and {\tt MP} tags.}), and a base modification symbol.
+This is then followed by a comma separated list of how many unmodified seq bases of the
+stated base type to skip, stored as a delta to the last and starting
+with 0 as the first (or next) base.  Hence this number series is
+comparable to the numbers in an {\tt MD} tag.
+
+For example {\tt C+m,5,12,0;} tells us there are three 5-Methylcytosine
+bases in the original {\sf SEQ}.  The first 5 {\tt C} bases are
+unmodified and the 6th is modified, as are the 19th (12 between the
+6th and 19th) and 20th.  Similarly {\tt G-m,14;} indicates the 15th
+{\tt G} is a 5-Methylcytosine on the opposite strand.
+
+This permits modifications to be listed on either strand with the rare
+potential for both strands to have a modification at the same site.
+
+Note it is permitted for the coordinate list to be empty (for example {\tt MM:Z:C+m;}), which may be used as an explicit indicator that this base modification is not present.
+It is not permitted for coordinates to be beyond the length of the sequence.
+
+If the modification is not one of the standard common types (listed
+below) it can be specified as a numeric ChEBI code.  For example
+{\tt C+76792,57;} is the same as {\tt C+h,57;}.
+
+An unmodified base of {\tt N} means count any base in {\sf SEQ}, not
+only those of {\tt N}.  Thus {\tt N+n,100;} means the 101st base is
+Xanthosine (n), irrespective of the sequence composition.
+
+The standard code types and their associated ChEBI values are listed
+below, taken from \emph{Modeling methyl-sensitive transcription factor
+motifs with an expanded epigenetic alphabet}, Coby Viner
+et.al. \url{https://www.biorxiv.org/content/10.1101/043794v1}.
+Additionally ambiguity codes {\tt A}, {\tt C}, {\tt G} and {\tt T}
+exist to represent unspecified modifications bases of their respective
+canonical base types, plus code {\tt N} to represent an unspecified
+modification of any base type.
+
+\begin{center}
+\begin{tabular}{lllll}
+{\bf Unmodified base} & {\bf Code} & {\bf Abbreviation} & {\bf Name} & {\bf ChEBI} \\
+\hline
+C & m & 5mC   & 5-Methylcytosine        & 27551 \\
+C & h & 5hmC  & 5-Hydroxymethylcytosine & 76792 \\
+C & f & 5fC   & 5-Formylcytosine        & 76794 \\
+C & c & 5caC  & 5-Carboxylcytosine      & 76793 \\
+C & C &       & Ambiguity code; any C mod & \\
+\hline
+T & g & 5hmU  & 5-Hydroxymethyluracil   & 16964 \\
+T & e & 5fU   & 5-Formyluracil          & 80961 \\
+T & b & 5caU  & 5-Carboxyluracil        & 17477 \\
```

Comment:
probably the same thing (IINAC) but on chebi this is 'uracil-5-carboxylic acid'

Reply:
It's great to see this being incorporated!

I would suggest keeping the "l" for all "carboxyl" prefixes. ChEBI often makes use of names that are uncommon within the applied areas in which the compounds are referenced, as in this case. I've never seen any modified base literature not use the "l".

I've checked the above-mentioned ChEBI IDs and all are correct. You can also find a catalogue of these on our [DNAmod webpage](https://dnamod.hoffmanlab.org), which contains these same recommendations and also lists the ChEBI IDs near the bottom (e.g. [5hmC](https://dnamod.hoffmanlab.org/5-(hydroxymethyl)cytosine.html)).

Hope that helps—really appreciate all the work being done on this!